test/plugins/individual-collateral/num/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { bn, fp } from '../../../../common/numbers'
+import { networkConfig } from '../../../../common/configuration'
+
+// oracle settings
+export const ORACLE_FEED = networkConfig['8453'].chainlinkFeeds.nARS!
+export const ORACLE_TIMEOUT = bn('86400')
+export const ORACLE_ERROR = fp('0.0025')
```

Comment:
different from what is in the deployment scripts (1%)

Reply:
Fixed thanks!